pyOCD/tools/flash_tool.py

```diff
@@ -99,6 +99,7 @@ def int_base_0(x):
 parser.add_argument("-fp", "--fast_program", action="store_true",
                     help="Use only the CRC of each page to determine if it already has the same data.")
 parser.add_argument("-da", "--daparg", dest="daparg", nargs='+', help="Send setting to DAPAccess layer.")
+parser.add_argument("--mass-erase", action="store_true", help="Mass erase the target device. Only for Kinetis devices.")
```

Comment:
Yes, certainly. But currently no other target supports a mass erase API. I can add an API to the `Target` class if you like. Maybe even with a default implementation that just uses the flash algo to do a chip erase.